### PR TITLE
genre list added!

### PR DESCRIPTION
**Pull request by sabrina34352**

### CODE REVIEW

**Review by IvanUglovets — APPROVED** [review]

File: packages/frontapp/src/app/components/DonateBook/index.tsx [hunk 1]
```diff
@@ -221,6 +222,23 @@ const StyledButton = styled(Button)`
...
+  margin-bottom: ${dimensions.base};
+  background: ${colors.bg_secondary};
+  :hover {
+    border: 0.5px solid ${colors.accent_color};
+  }
+
+  :focus {
+    border: 0.5px solid ${colors.accent_color};
+  }
+`;
```
Comment (x8er) [inline packages/frontapp/src/app/components/DonateBook/index.tsx:241]: We have drop down list already

**Review by x8er — CHANGES_REQUESTED** [review]
Also fix github warnings pls

**Review by x8er — CHANGES_REQUESTED** [review]
fix the warnings pls

**Comment by x8er** [comment]
@sabrina34352 Okay

File: packages/frontapp/src/app/components/ListOfGenres/index.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  },
+  {
+    id: 17,
+    value: 'Art',
+  },
+  {
+    id: 18,
+    value: 'Other',
+  },
+];
```
Comment (x8er) [inline packages/frontapp/src/app/components/ListOfGenres/index.ts:74]: this is not a component

**Review by x8er — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/ListOfGenres/index.ts (hunk 2 above)
Comment (sabrina34352) [reply packages/frontapp/src/app/components/ListOfGenres/index.ts:74]: it is used in two places, whats the better way to code this?

Comment (x8er) [reply packages/frontapp/src/app/components/ListOfGenres/index.ts:74]: add file which name as `consts.ts` or something like that in `assets` folder

**Review by x8er — APPROVED** [review]

**Review by x8er — APPROVED** [review]

**Review by IvanUglovets — APPROVED** [review]

**Pull request merged**